### DISCUSSION TITLE
Replace node with bun

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#16320] [General - Components / Directives / etc] [answered]
Now that bun is at version 1.0 (https://bun.sh/blog/bun-v1.0) I was curious how difficult it would be to migrate a quasar app. Has anyone tried? Is this possible?

**Comment by ssuess** [comment]
I did some initial poking around, and at least in my case it really does seem (seem?) to be a drop-in replacement. Steps I followed:
1. Install bun (`curl -fsSL https://bun.sh/install | bash`)
2. install quasar cli (`bun install -g @quasar/cli`)
3. copy folder with existing quasar project to new folder, remove node_modules folder
4. cd into new folder, install (`bun install`)
5. run project with `quasar dev`

RESULTS:
1. `bun install` was very fast compared to `npm install`
2. not sure if it should be any different here tbh, but `quasar build` was identical in new and old environments

**Comment by rstoenescu** [comment, marked as answer]
Bun support coming to q/app-vite v1.6 and q/app-webpack 3.11.
The work has been backported from the next major versions of these CLIs (currently unreleased to the public).

**Reply by ssuess** [reply]
Thanks so much for this! Does this only affect package installs, or will quasar be using bun.serve() or any other part of bun?
